libjwt/jwt.c

```diff
@@ -1094,6 +1094,24 @@ static int jwt_dump(jwt_t *jwt, char **buf, int pretty)
 	return ret;
 }
 
+char* jwt_dump_grants_str(jwt_t* jwt, int pretty) 
```

Comment:
Following convention in the rest of the code, make sure `*` is aligned with the argument, not the type: I.e.

    char* jwt_dump_grants_str(jwt_t *jwt, int pretty)